src/config.js

```diff
@@ -4,9 +4,15 @@ import cornerstoneWADOImageLoader from 'cornerstone-wado-image-loader';
 import dicomParser from 'dicom-parser';
 import version from './version.js';
 
+let homepage;
+const { process } = window;
+if (process && process.env && process.env.PUBLIC_URL) {
+  homepage = process.env.PUBLIC_URL;
+}
+
 window.info = {
```

Comment:
If we're attaching anything to the window with script-tag, you may want to vendor prefix:
`$ohif.info`

If only to reduce chances of conflict

Reply:
This doesn't change anything we are attaching, it just changes how `process` is checked, since it is undefined sometimes.

We need to get away from using `window.config` entirely and use Context or Redux to pass that data around. This can be another PR though.